tests/Core/RouterTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+// use PHPUnit\Framework\TestCase;
+use TrendingRepos\Core\Router;
+use TrendingRepos\Core\Request;
+use TrendingRepos\Exception\RouterException;
+use Tests\TestCase;
+
+class RouterTest extends TestCase
+{
+    protected $router;
+    protected $client;
+
+    public function setUp(): void
+    {
+        $this->router = new Router([], new Request);
+    }
+
+    public function test_init()
+    {
+        $this->assertInstanceOf('\\TrendingRepos\\Core\\Router', $this->router);
+    }
```

Comment:
This test case is unnecessary, result of `new Router` will always be an instance of `Router` or the fatal PHP error.